packages/@ourworldindata/grapher/src/controls/Controls.scss

```diff
@@ -1,175 +1,408 @@
+
+    button.menu-toggle {
+        font: $medium 13px/1.3 $lato;
+        letter-spacing: 0.01em;
+        display: flex;
+        align-items: center;
+        border: 1px solid $light-stroke;
+        border-radius: 4px;
+        padding: 7px;
+        color: $dark-text;
+
+        &:hover {
+            background: $hover-fill;
+            cursor: pointer;
+        }
+
+        &.active,
+        &:active {
+            background: $active-fill;
+            border: 1px solid $active-fill;
+        }
+
+        svg {
+            height: 12px;
+            margin-right: 5px;
+            vertical-align: bottom;
+        }
     }
-}
-
-.FacetStrategyLabel {
-    position: static;
-    left: 4.17%;
-    right: 44.79%;
-    top: 5.88%;
-    bottom: 64.71%;
-
-    font-family: Lato;
-    font-style: normal;
-    font-weight: bold;
-    font-size: 12px;
-    line-height: 1.2;
 
-    margin-top: 4px;
-}
-
-.selected .FacetStrategyLabel {
-    color: #0073e6;
-}
-.FacetStrategyLabel {
-    color: #7d899c;
-}
-
-.FacetStrategyPreview-none-child {
-    position: absolute;
-    left: 0%;
-    right: 0%;
-    top: 0%;
-    bottom: 0%;
-
-    border-radius: 1px;
-}
-
-.FacetStrategyPreview-parent {
-    position: relative;
-    width: 100%;
-    height: 36px;
-}
-
-.FacetStrategyPreview-split-child {
-    float: left;
-    width: 28px;
-    height: 17px;
-    border-radius: 1px;
-}
-
-// set appropriate margins for our manual 3x2 grid
-.FacetStrategyPreview-split-child:nth-of-type(1),
-.FacetStrategyPreview-split-child:nth-of-type(2),
-.FacetStrategyPreview-split-child:nth-of-type(3) {
-    margin-bottom: 2px;
-}
-
-.FacetStrategyPreview-split-child:nth-of-type(1),
-.FacetStrategyPreview-split-child:nth-of-type(2),
-.FacetStrategyPreview-split-child:nth-of-type(4),
-.FacetStrategyPreview-split-child:nth-of-type(5) {
-    margin-right: 2px;
-}
-
-.selected .FacetStrategyPreview-none-child {
-    background: #d9eafb;
-}
-
-.FacetStrategyPreview-none-child {
-    background: #dde1e7;
+    .settings-menu {
+        // the pop-up version of the settings menu
+        .settings-menu-controls {
+            position: absolute;
+            width: 300px;
+            right: $indent;
+            background: white;
+            border-radius: 4px;
+            box-shadow: 0px 4px 23px 4px #0000000f;
+            z-index: $zindex-controls-drawer;
+            overflow-y: scroll;
+        }
+
+        // transparent mouse-event catcher while popup is visible
+        .settings-menu-backdrop {
+            position: absolute;
+            left: 0;
+            top: 0;
+            width: 100%;
+            height: 100%;
+            z-index: $zindex-controls-backdrop;
+        }
+    }
 }
 
-.selected .FacetStrategyPreview-split-child {
-    background: #d9eafb;
+@at-root {
+    // collapse the Settings toggle down to just an icon on small screens
+    .GrapherComponentSmall nav.controlsRow .settings-menu button.menu-toggle {
+        min-height: 32px;
```

Comment:
You could be using `$controlRowHeight` here if it's available. If it's not, we could consider moving the defintion elsewhere (it's currently defined in `CaptionedChart.scss`)